static/css/impala/users.less

```diff
@@ -321,3 +321,7 @@ img.icon {
 #my-themes .view-all {
     float: right;
 }
+
+.html-rtl #my-themes .view-all {
```

Comment:
In the future, because it's less, I think you could do:

```less
 #my-themes .view-all {
    float: right;

    .html-rtl & {
         float: left;
    }
 }
```

But this is fine 😄